website/src/css/_typography.scss

```diff
@@ -0,0 +1,22 @@
+@import '@styles/colors';
+
+// Custom font weights
+$font-weights: 500, 550, 600, 700;
+
+// Custom font weight utility classes
+@each $font-weight in $font-weights {
+    .fw-#{$font-weight} {
+        font-weight: #{$font-weight};
+    }
+}
+
+$colors: (
```

Comment:
Would love to hear @bretthayes's thoughts on this

Reply:
Our colours need a complete overhaul 😅  I was planning to tackle this in the replatform, but we seem to be needing a lot more simplicity here with utility classes, vs creating more stylesheets while we're tackling more marketing requests in tandem. 😬

Reply:
Nice!